build.gradle

```
@@ -488,3 +489,10 @@ generateCCDConfig.doLast {
     into layout.buildDirectory.dir('definitions/NFD')
   }
 }
+
+task fortifyScan(type: JavaExec) {
+    main = 'uk.gov.hmcts.fortifyclient.FortifyClientMainApp'
```

mainClass is the newer syntax recommended by Gradle:
```suggestion
    mainClass = 'uk.gov.hmcts.fortifyclient.FortifyClientMainApp'
```